src/Filter/DateComp.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import { DatePicker } from "antd";
+
+const DateComp = () => {
+  return (
+    <div className="date-row">
+      <div className="date-first-col">
+        <p>From</p>
+        <DatePicker bordered={true} placeholder="Date" style={{ width: 110 }} />
+      </div>
+      <div className="date-second-col">
+        <p>To</p>
+        <DatePicker
+          picker="date"
+          bordered={true}
+          placeholder="Date"
+          style={{ width: 110 }}
+        />
+      </div>
+    </div>
```

Comment:
can you please use a rangepicker instead 
https://3x.ant.design/components/date-picker/#RangePicker

Reply:
Changed the RangePicker.